pydash/numerical.py

```diff
@@ -383,6 +386,43 @@ def std_deviation(array):
     return math.sqrt(variance(array))
 
 
+def call_math_operator(value1, value2, callback):
+    """Return the result of the math operation on the given values.
+
+    Args:
+        value1 (int/float/str): Value passed by the user.
+        value2 (int/float/str): Value passed by the user.
+        callback (mixed, optional): Callback applied on the given values.
+
+    Returns:
+        float: Result after the math operation is conducted on the given
+            values.
+
+    Example:
+
+        >>> call_math_operator(5, 4, lambda x, y: x - y)
+        1.0
+        >>> call_math_operator(4, 6, lambda x, y: x * y)
+        24.0
+        >>> call_math_operator('5', '10', lambda x, y: x + y)
+        15.0
+        >>> call_math_operator('25', '5', lambda x, y: x / y)
+        5.0
+
+    .. versionadded:: TODO
+    """
```

Comment:
Helper functions don't get this level of detail in the docstrings. They're not meant for public use so only really need a brief description. No need for args, returns, or example sections.